docker/services.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.9'
+
+
+services:
+  mysql_db:
+    image: ${MYSQL_VERSION}
+    container_name: db
+    ports:
+      - 3306:3306
+    expose:
+      - 3306
+    env_file: ./db/.env
+    restart: always
+    volumes:       
+      - db:/var/lib/mysql
+      - ./db/init.sql:/docker-entrypoint-initdb.d/init.sql
```

Comment:
Maybe do a copy of the default `/node_modules/reldens/migrations/production/reldens-install-v4.0.0.sql` as `init.sql`?
or just wget from https://github.com/damian-pastorini/reldens/blob/master/migrations/production/reldens-install-v4.0.0.sql? something I could update with each new version.

Thanks!

Reply:
Hi @damian-pastorini . I have added the initialization script to docker/migrations directory so mysql can pick the script at runtime. I did test the configuration and it runs but there is a bug with registering authentication routes which doesn't let user go beyond home page.

Reply:
could you walk me a bit more through that issue? you mean an issue in the skeleton or with some part of the platform? 
normally, if the server starts but you can pass the login or register a new user, is probably because some previous issue on the client that's normally visible in the browser console

Reply:
hi @Rilord I'm trying to take a look on this, but my bit of docker knowledge is not helping me :P sorry, what's the procedure to create the local setup using the docker folder? Or better ask: which are the commands to create and run the containers?
maybe a good idea would be to include some notes in the README.md about it?
Thanks!

Reply:
greetings @damian-pastorini I've updated the [README](https://github.com/Rilord/reldens-skeleton/tree/docker-scripts#beta-running-inside-docker) file, so you will be able to run the app with little edits to dotfiles. Please report your bugs if there any of course.

Reply:
hi @Rilord , thanks and sorry for the late reply.
today I was giving it another try and I'm getting the followin error:
```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 704, in urlopen
    httplib_response = self._make_request(
                       ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 399, in _make_request
//....
```
does this rings any bells to you?

thanks!